### PR TITLE
release(core): prepare release notes

### PR DESCRIPTION
**Pull request by cgagnaire**
## Description

Prepare release notes for core softwares.

## Target serie

- [ ] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

File: en/releases/centreon-core.md [hunk 1]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+### Enhancements
+
+- [API/Core/Configuration] Log actions when adding/enabling/disabling hosts
+- [API] Possibility to Register servers (Remote Server, Poller, Centreon Map)
+- [Configuration/Wizard] Add possibility to select registered poller
+- [Authentication] Replace Keycloak to generic OAuth2 / OpenId Connect
+- [Event Logs] Display anomaly detection as regular service
+- [Monitoring/Resources Status] Add shortcuts for hosts and services details
+- [Monitoring/Resources Status] Add timeline for hosts and services details
+- [Monitoring/Resources Status] Be alble to filter on status output
```
Comment (kduret) [inline en/releases/centreon-core.md:32]:
```suggestion
- [Monitoring/Resources Status] Be able to filter on status output
```

File: fr/releases/centreon-core.md [hunk 2]
```diff
@@ -5,562 +5,110 @@ title: Centreon Core
...
-* Update composer dependencies (PR/#8515)
-* Upgrade handlebars dependencies (PR/#8224)
+- [API/Core/Configuration] Log actions when adding/enabling/disabling hosts
+- [API] Possibility to Register servers (Remote Server, Poller, Centreon Map)
+- [Configuration/Wizard] Add possibility to select registered poller
+- [Authentication] Replace Keycloak to generic OAuth2 / OpenId Connect
+- [Event Logs] Display anomaly detection as regular service
+- [Monitoring/Resources Status] Add shortcuts for hosts and services details
+- [Monitoring/Resources Status] Add timeline for hosts and services details
+- [Monitoring/Resources Status] Be alble to filter on status output
```
Comment (kduret) [inline fr/releases/centreon-core.md:32]:
```suggestion
- [Monitoring/Resources Status] Be able to filter on status output
```

**Review by Thebarda — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+>     communication will not be readable by Engine process, resulting to this
+>     error in Engine log:
+>
+>     ```text
+>     Error: Parsing of global configuration failed: Can't open file '/etc/centreon-engine/centengine.cfg'
+>     ```
+>
+>     This is due to more strict rights on those files.
+>
+>     As always, we **strongly recommand** to upgrade all components to
```
Comment (chgautier) [inline en/releases/centreon-core.md:55]:
```suggestion
>     As always, we **strongly recommend** to upgrade all components to
```

**Review by lpinsivy — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 4]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+>     match Central serie.
```
Comment (chgautier) [inline en/releases/centreon-core.md:56]:
```suggestion
>     match the Central server's version.
```

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+>
+>     However, during an upgrade process, it is possible to manually
+>     add the user *centreon-engine* to *centreon-gorgone* group with
```
Comment (chgautier) [inline en/releases/centreon-core.md:59]:
```suggestion
>     add the user *centreon-engine* to the *centreon-gorgone* group with
```

File: en/releases/centreon-core.md [hunk 6]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
-
-> Known issues:
+> Known behaviours:
 >
-> - When updating to 20.04.5, some badly encoded characters might block SQL
->   Broker events processing on the Central server. We strongly recommand to
->   update all the Engines to 20.04.3 version while updating to this Broker
->   version.
+>   - If Broker on a Poller or Remote Server is not upgraded to 20.10
+>     (or with a version prior to 20.04.9), the communication between those
```
Comment (chgautier) [inline en/releases/centreon-core.md:77]:
```suggestion
>     (or with a version prior to 20.04.9), the communication between said
```

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+>     Poller or Remote Server and an upgraded Central may not work.
 >
->   If you experience this behaviour, we recommand to downgrade Broker on the
->   Central server using this command:
+>     As always, we **strongly recommand** to upgrade all components to match
```
Comment (chgautier) [inline en/releases/centreon-core.md:80]:
```suggestion
>     As always, we **strongly recommend** to upgrade all components to match
```

File: en/releases/centreon-core.md [hunk 8]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+>     Central serie.
```
Comment (chgautier) [inline en/releases/centreon-core.md:81]:
```suggestion
>     the Central server's version.
```

File: en/releases/centreon-core.md [hunk 9]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
 >
->   ```shell
->   yum downgrade centreon-broker-20.04.4 centreon-broker-cbd-20.04.4 centreon-broker-storage-20.04.4 centreon-broker-cbmod-20.04.4 centreon-broker-core-20.04.4
->   ```
+>     However, during an upgrade process, communication can be maintained
+>     by making sure Brokers' configurations match the following conditions:
```
Comment (chgautier) [inline en/releases/centreon-core.md:84]:
```suggestion
>     by making sure Broker's configurations match the following conditions:
```

File: fr/releases/centreon-core.md [hunk 10]
```diff
@@ -5,562 +5,110 @@ title: Centreon Core
...
+Vous trouverez dans ce chapitre tout ce qui concerne **Centreon Core**.
 
-> It is very important when you update your system to refer to this section in
-> order to learn about behavior changes or major changes that have been made on
-> this version. This will let you know the impact of the installation of these
-> versions on the features you use or the specific developments that you have
-> built on your platform (modules, widgets, plugins).
+> Il est important de mettre à jour en utilisant la documentation adéquate de
+> mise à jour et de lire attentivement les notes de mise à jour afin d'être au
+> courant de changements qui pourraient impacter votre usage ou votre plateforme
```
Comment (chgautier) [inline fr/releases/centreon-core.md:12]:
```suggestion
> courant des changements qui pourraient impacter votre usage ou votre plateforme
```

File: fr/releases/centreon-core.md [hunk 11]
```diff
@@ -5,562 +5,110 @@ title: Centreon Core
...
+>   - Si Engine sur un Poller ou un Remote Server n'est pas mis à jour en
+>     20.10, les fichiers de configuration copiés depuis un Central à jour
+>     par Gorgone utilisant une communication ZMQ ne pourront pas être
+>     lu par le processus Engine, avec l'erreur suivante dans les logs:
+>
+>     ```text
+>     Error: Parsing of global configuration failed: Can't open file '/etc/centreon-engine/centengine.cfg'
+>     ```
+>
+>     Cela est dû a des droits plus strict sur ces fichiers.
```
Comment (chgautier) [inline fr/releases/centreon-core.md:53]:
```suggestion
>     Cela est dû à des droits plus stricts sur ces fichiers.
```

File: fr/releases/centreon-core.md [hunk 12]
```diff
@@ -5,562 +5,110 @@ title: Centreon Core
...
-#### Bug fixes
-
-- Fix macros replacement
-- Fix perfdata truncated when retention read
+> Comportements connus:
+>
+>   - Si Engine sur un Poller ou un Remote Server n'est pas mis à jour en
+>     20.10, les fichiers de configuration copiés depuis un Central à jour
+>     par Gorgone utilisant une communication ZMQ ne pourront pas être
+>     lu par le processus Engine, avec l'erreur suivante dans les logs:
```
Comment (chgautier) [inline fr/releases/centreon-core.md:47]:
```suggestion
>     lus par le processus Engine, avec l'erreur suivante dans les logs:
```

File: fr/releases/centreon-core.md (hunk 11 above)
Comment (chgautier) [inline fr/releases/centreon-core.md:53]:
```suggestion
>     Cela est dû à des droits plus stricts sur ces fichiers.
```

File: fr/releases/centreon-core.md [hunk 13]
```diff
@@ -5,562 +5,110 @@ title: Centreon Core
...
->   If you experience this behaviour, we recommand to downgrade Broker on the
->   Central server using this command:
+>     Comme toujours, nous **recommandons fortement** de mettre à jour
+>     tous les composants en adéquation avec la version du Central.
 >
->   ```shell
->   yum downgrade centreon-broker-20.04.4 centreon-broker-cbd-20.04.4 centreon-broker-storage-20.04.4 centreon-broker-cbmod-20.04.4 centreon-broker-core-20.04.4
->   ```
+>     Cependant, le temps de la mise à jour, la communication peut être
+>     maintenu en s'assurant que les configurations des Brokers respectent
```
Comment (chgautier) [inline fr/releases/centreon-core.md:85]:
```suggestion
>     maintenue en s'assurant que les configurations de Broker respectent
```

File: en/releases/centreon-core.md [hunk 14]
```diff
@@ -18,549 +18,95 @@ If you have feature requests or want to report a bug, please go to our
...
+>   - If Engine on a Poller or Remote Server is not upgraded to 20.10,
+>     configuration files copied from an upgraded Central by Gorgone using ZMQ
+>     communication will not be readable by Engine process, resulting to this
+>     error in Engine log:
+>
+>     ```text
+>     Error: Parsing of global configuration failed: Can't open file '/etc/centreon-engine/centengine.cfg'
+>     ```
+>
+>     This is due to more strict rights on those files.
```
Comment (chgautier) [inline en/releases/centreon-core.md:53]:
```suggestion
>     This is due to stricter rights on those files.
```

File: en/releases/centreon-core.md (hunk 3 above)
Comment (chgautier) [inline en/releases/centreon-core.md:55]:
```suggestion
>     As always, we **strongly recommend** to upgrade all components to
```

File: en/releases/centreon-core.md (hunk 4 above)
Comment (chgautier) [inline en/releases/centreon-core.md:56]:
```suggestion
>     match the Central  server's version.
```

File: en/releases/centreon-core.md (hunk 5 above)
Comment (chgautier) [inline en/releases/centreon-core.md:59]:
```suggestion
>     add the user *centreon-engine* to the *centreon-gorgone* group with
```

File: en/releases/centreon-core.md (hunk 6 above)
Comment (chgautier) [inline en/releases/centreon-core.md:77]:
```suggestion
>     (or with a version prior to 20.04.9), the communication between said
```

File: en/releases/centreon-core.md (hunk 7 above)
Comment (chgautier) [inline en/releases/centreon-core.md:80]:
```suggestion
>     As always, we **strongly recommend** to upgrade all components to match
```

File: en/releases/centreon-core.md (hunk 8 above)
Comment (chgautier) [inline en/releases/centreon-core.md:81]:
```suggestion
>     the Central server's version.
```

File: en/releases/centreon-core.md (hunk 9 above)
Comment (chgautier) [inline en/releases/centreon-core.md:84]:
```suggestion
>     by making sure Broker's configurations match the following conditions:
```

File: fr/releases/centreon-core.md (hunk 10 above)
Comment (chgautier) [inline fr/releases/centreon-core.md:12]:
```suggestion
> courant des changements qui pourraient impacter votre usage ou votre plateforme
```

**Review by chgautier — CHANGES_REQUESTED** [review]

File: fr/releases/centreon-core.md [hunk 15]
```diff
@@ -5,562 +5,110 @@ title: Centreon Core
...
```
Comment (cgagnaire) [inline fr/releases/centreon-core.md:85]:
```suggestion
>     maintenue en s'assurant que les configurations des Brokers respectent
```

File: fr/releases/centreon-core.md (hunk 13 above)
Comment (cgagnaire) [reply fr/releases/centreon-core.md]: There is a Broker on both Central and Poller or Remote Server, either a dedicated service or a Engine's module.

**Review by chgautier — APPROVED** [review]

**Pull request merged**